src/wp-includes/post.php

```diff
@@ -7660,42 +7658,17 @@ function wp_queue_posts_for_term_meta_lazyload( $posts ) {
 					// Backward compatibility for if a plugin is putting objects into the cache, rather than IDs.
 					foreach ( $_term_ids as $term_id ) {
 						if ( is_numeric( $term_id ) ) {
-							$prime_term_ids[]                  = (int) $term_id;
-							$prime_taxonomy_ids[ $taxonomy ][] = (int) $term_id;
+							$term_ids[] = (int) $term_id;
 						} elseif ( isset( $term_id->term_id ) ) {
-							$prime_taxonomy_ids[ $taxonomy ][] = (int) $term_id->term_id;
-							$prime_term_ids[]                  = (int) $term_id->term_id;
+							$term_ids[] = (int) $term_id->term_id;
 						}
 					}
 				}
 			}
 		}
-
-		if ( $prime_term_ids ) {
-			$prime_term_ids = array_unique( $prime_term_ids );
-			// Do not prime term meta at this point, let the lazy loader take care of that.
-			_prime_term_caches( $prime_term_ids, false );
```

Comment:
So before, we were doing a call to `get_term` to check it exists. But it is extremely unlikely the term will not exist, making an extensive call to `get_term` unneeded. This prime of terms was only done because of this. The term does not need to be loaded here. 99% of the time, the term is already in memory anyway, making this call wasteful. To prime term meta, all we need is the id.

Reply:
CC @joemcgill

Reply:
This change is somewhat unrelated, I will revert it for now.